src/components/Tables/ResourceAvailabilityTable.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <div>
+    <div v-if="resources.length === 0">
+      <p class="no-resources">No resources available.</p>
+    </div>
+    <div v-else>
+      <div class="table">
+        <table class="table-table">
+          <thead class="table-component">
+            <tr>
+              <th>Owner Name</th>
+              <th>Resource Type</th>
+              <th>Resource Class</th>
+              <th>Resource Color</th>
+              <th>Quantity</th>
+            </tr>
+          </thead>
+          <tbody class="table-component">
+            <tr v-for="(resource, index) in resources" :key="index">
+              <td>{{ resource.owner.name }}</td>
+              <td>{{ resource.resources[index].type }}</td>
+              <td>{{ resource.resources[index].clazz }}</td>
+              <td>{{ resource.resources[index].color }}</td>
+              <td>{{ resource.quantity }}</td>
```

Comment:
I realise we haven't mentioned which fields to include in the rows but we might want to include all of them just like in the resources table. It's going to be overly verbose but we haven't thought of how to show enough information so the user knows which resource is meant to be shown and also to not show too much information so that they get lost in the data.
If possible some grouping of the columns might be a good idea. All the resource information columns to be in a group and the quantityType and quantity in another group.

Reply:
We are going to do that after the vuetify merge

Reply:
Actually, I made it

Reply:
Done